articles/quickstart/spa/_includes/_authz_handle_scopes.md

```diff
@@ -0,0 +1,3 @@
+## Handle Scopes in the ${service} service
+
+Adjust your ${service} service to use a local member with any scopes you want to request when users log in. Use this member in your instance of the `auth0.WebAuth` object.
```

Comment:
@aaguiarz, could you help me understand what this sentence means? It's not clear so I'm not sure how can I correct it. Is it: 
Adjust the service with scopes, so it can use a local member
or
Adjust the service, so it uses a local member with scopes